src/System.Net.Mail/src/System/Net/Mail/NTAuthenticationStub.cs

```diff
@@ -0,0 +1,96 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Reflection;
+using System.Net.Security;
+using System.Security.Authentication.ExtendedProtection;
+using System.Diagnostics;
+
+namespace System.Net
+{
+    internal class NTAuthentication
```

Comment:
Just curious: Is this stub temporary due to layering issues that will be resolved before shipping, or is it going to ship this way?